phoenix-core/src/it/java/org/apache/phoenix/end2end/AlterTableWithViewsIT.java

```diff
@@ -107,12 +107,14 @@ public void testAddNewColumnsToBaseTableWithViews() throws Exception {
             assertTableDefinition(conn, tableName, PTableType.TABLE, null, 0, 3, QueryConstants.BASE_TABLE_BASE_COLUMN_COUNT, "ID", "COL1", "COL2");
             
             viewConn.createStatement().execute("CREATE VIEW " + viewOfTable + " ( VIEW_COL1 DECIMAL(10,2), VIEW_COL2 VARCHAR ) AS SELECT * FROM " + tableName);
-            assertTableDefinition(conn, viewOfTable, PTableType.VIEW, tableName, 0, 5, 3, "ID", "COL1", "COL2", "VIEW_COL1", "VIEW_COL2");
+            assertTableDefinition(conn, viewOfTable, PTableType.VIEW, tableName, 0, 5, 3, "VIEW_COL1", "VIEW_COL2");
```

Comment:
Sure we can do that, I think we will need to modify a ton of tests to make this happen.  Its not only done here. 

Another thing is that there are a few tests where the assumptions of how the system works has changed.  A few examples are tests like this: 

ViewIT.testDisallowDropOfColumnOnParentTable()
ViewIT.testDisallowDropOfReferencedColumn()
QueryCompiler.testDuplicatePKColumn() 
(this one because we have to support both the diff based approach and the current approach for our migration step, we remove duplicate columns on read, thus we don't prevent this from happening). 

There are quite a few more, we can maybe discuss if these tests are no longer valid or how we can change them to make them meaningful.